CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 * Loosen up `Test_LBAppRule` for invalid application script check to work with different error engine in VCD 10.2
 [#326](https://github.com/vmware/go-vcloud-director/pull/326)
 * Update VDC dynamic func to handle API version 35.0 [#327](https://github.com/vmware/go-vcloud-director/pull/327)
+* Introduce low level OpenAPI client functions `OpenApiGetAllItems`,`OpenApiPostItemSync`,`OpenApiPostItemAsync`,
+`OpenApiPostItem`, `OpenApiGetItem`, `OpenApiPutItem`, `OpenApiPutItemSync`, `OpenApiPutItemAsync`,
+`OpenApiDeleteItem`, `OpenApiIsSupported`, `BuildOpenApiEndpoint`
```

Comment:
I think you renamed to `OpenApiBuildEndpoint`, but there are `BuildOpenApiEndpoint` occurrences in the PR docs still.

Reply:
Sorry, indeed. I got over optimistic about refactoring...